commands/user_e2e_test.go

```diff
@@ -6,11 +6,12 @@ package commands
 import (
 	"fmt"
 
+	"github.com/hashicorp/go-multierror"
 	"github.com/mattermost/mattermost-server/v6/model"
-	"github.com/spf13/cobra"
-
 	"github.com/mattermost/mmctl/v6/client"
 	"github.com/mattermost/mmctl/v6/printer"
+	"github.com/spf13/cobra"
+	"net/http"
```

Comment:
This should be moved up to the standard packages.

Reply:
fixed imports // make build which also runs the linter says things are fine now